demos/python_demos/object_detection_demo_yolov3_async/object_detection_demo_yolov3_async.py

```diff
@@ -310,10 +339,12 @@ def main():
         presenter.drawGraphs(frame)
         for obj in objects:
             # Validation bbox of detected object
-            if obj['xmax'] > origin_im_size[1] or obj['ymax'] > origin_im_size[0] or obj['xmin'] < 0 or obj['ymin'] < 0:
-                continue
+            obj['xmax'] = min(obj['xmax'], origin_im_size[1])
+            obj['ymax'] = min(obj['ymax'], origin_im_size[0])
+            obj['xmin'] = max(obj['xmin'], 0)
+            obj['ymin'] = max(obj['ymin'], 0)
             color = (int(min(obj['class_id'] * 12.5, 255)),
-                     min(obj['class_id'] * 7, 255), min(obj['class_id'] * 5, 255))
+                     int(min(obj['class_id'] * 7, 255)), int(min(obj['class_id'] * 5, 255)))
```

Comment:
I can see why these `int`s are here, but it's not obvious at all. IMO, you should instead convert the class ID to `int` _before_ you put it into `obj` (and explain in a comment why you're doing it).

Reply:
I see that you removed the `int` casts. Were they not necessary after all? I thought they were needed to ensure that we pass `int` objects to OpenCV, rather than `np.int32`/`np.float64`?

Reply:
I moved `int` cast to parsing function since it should return `class_id` as int.

Reply:
Ah, I see now, you've used the `item` method to convert the value to a native Python type. Yeah, that's much better, thanks.